torch/functional.py

```diff
@@ -786,9 +796,10 @@ def chain_matmul(*matrices):
 
 
 def lu(A, pivot=True, get_infos=False, out=None):
-    r"""Computes the LU factorization of a square matrix or batches of square matrices
-    :attr:`A`. Returns a tuple containing the LU factorization and pivots of :attr:`A`.
-    Pivoting is done if :attr:`pivot` is set to ``True``.
+    r"""Computes the LU factorization of a rectangular matrix or batches of
```

Comment:
rectangular matrix seems redundant, why can't we just specify ```matrix``` or ```matrices`` instead?

Reply:
Yes, removed `rectangular`.